tools/hive-bigquery/utilities.py

```diff
@@ -0,0 +1,89 @@
+"""Provides utility functions to other modules"""
+
+import logging
+import random
+import string
+import subprocess
+
+import tableprint as tp
+
+logger = logging.getLogger('Hive2BigQuery')
+
+
+def calculate_time(start, end):
+    """Pretty prints the time taken for an operation
+
+    Args:
+        start (float): Start time of an operation
+        end (float): End time of an operation
+
+    Returns:
+        str: Pretty format the time taken for an operation
+    """
+
+    time_taken = int(round((end - start), 0))
+    day = time_taken // 86400
+    hour = (time_taken - (day * 86400)) // 3600
+    minutes = (time_taken - ((day * 86400) + (hour * 3600))) // 60
+    seconds = time_taken - ((day * 86400) + (hour * 3600) + (minutes * 60))
+
+    if day != 0:
+        output = '{} days {} hours {} min {} sec'.format(day, hour, minutes,
+                                                         seconds)
+    elif hour != 0:
+        output = '{} hours {} min {} sec'.format(hour, minutes, seconds)
+    elif minutes != 0:
+        output = '{} min {} sec'.format(minutes, seconds)
+    else:
+        output = '{} sec'.format(seconds)
+
+    return output
+
+
+def print_and_log(output, level=logging.DEBUG):
```

Comment:
This function is redundant of a method the `logging` library provides you.
Please refactor so whenever you call:
`logging.getLogger()` simply add `logging.getLogger().addHandler(logging.StreamHandler)`
and use the defined logger object to achieve both logging and printing.

Reply:
print_and_log will pretty print the console logs which will help the user better identify what's going on.